src/server.cpp

```diff
@@ -426,6 +426,20 @@ CServer::CServer ( const int          iNewMaxNumChan,
         QThreadPool::globalInstance()->setMaxThreadCount ( QThread::idealThreadCount() * 4 );
     }
 
+    // initialize chat html sanitization patterns:
+    static QList<QString> qlPermittedChatTagNames = {
+        "b",
+        "i",
+        "p",
+        "pre",
+        "u",
+    };
+    foreach ( auto tag, qlPermittedChatTagNames )
+    {
+        QRegExp pattern = QRegExp ( "&lt;(" + tag + ")&gt;([^<>]+)&lt;/" + tag + "&gt;" );
+        pattern.setMinimal ( true ); // non-greedy matching
```

Comment:
Suggest it might be a good idea also to add the following:
```
pattern.setCaseSensitivity(Qt::CaseInsensitive);  // allow upper or lower case tags
```
Just in case the user it pasting in HTML that happens to use upper-case tags.